customising-the-view/index.md

```diff
@@ -119,7 +125,9 @@ And the same configuration in YAML:
 basePath: https://github.com/SassDoc/sassdoc
 shortcutIcon: assets/images/favicon.png
 display:
-    access: [public, private]
+    access:
+        - public
+        - private
```

Comment:
YAML indent is not consistent accross the website; sometimes 2 spaces, sometimes 4 (it was already the case, not a problem with this PR).

Maybe let's use 2 spaces for all YAML examples?